packages/fx-core/src/component/generator/openApiSpec/kiota.ts

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+/**
+ * @author yuqzho@microsoft.com, KennethBWSong, Ning Tang
```

Comment:
[Minor] Please also add renlong. You can update this in another pr.

Reply:
Thanks, I'll also update CODEOWNER in another PR.